cookbook/security/custom_authentication_provider.rst

```diff
@@ -596,23 +630,34 @@ set to any desirable value per firewall.
 
     .. code-block:: xml
 
-        <config>
-            <firewall name="wsse_secured"
-                pattern="/api/.*"
-            >
-                <stateless />
-                <wsse lifetime="30" />
-            </firewall>
-        </config>
+        <!-- app/config/security.xml -->
+        <?xml version="1.0" encoding="UTF-8"?>
+        <srv:container xmlns="http://symfony.com/schema/dic/security"
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xmlns:srv="http://symfony.com/schema/dic/services"
+            xsi:schemaLocation="http://symfony.com/schema/dic/services
+                http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+            <config>
+                <!-- ... -->
+
+                <firewall name="wsse_secured" pattern="/api/.*" stateless="true">
```

Comment:
In the examples above, we usually define the pattern as `^/api` instead of `/api/.*` Is there any reason for using this other format?